src/main/java/nl/novi/TechItEasy/dtos/RemoteControllerOutputDto.java

```diff
@@ -15,4 +15,6 @@ public class RemoteControllerOutputDto {
     private Double price;
     private Integer originalStock;
```

Comment:
Deze manier van Dto's gebruiken kende ik nog niet! Ik had hier geen constructor en dan voor elke variabele een getter en setter.

Reply:
Naast een lege constructor, kun je ook een constructor met alle parameters (of een gedeelte) maken. Dat kan handig zijn om in je mappers of straks in je tests te gebruiken. Gewoon met getters en setters werkt echter ook prima